modules/custom/az_person/config/install/views.view.az_reorder.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
     - draggableviews
     - node
     - user
+_core:
```

Comment:
This `_core:` key without a value looks like it might have been associated with a UUID value that was removed.  Maybe it should be removed (?)
```suggestion
```

Reply:
Yup that was meant to be removed. Thanks @joeparsons